### ISSUE TITLE
Unable to set an integer as the ng-panel id

### ISSUE THREAD

**Issue by grosch** [ng-bootstrap/ng-bootstrap#4066]
### Bug description:

ngb-panel id can't be a number.  If it is, then we get a failure from querySelector.  This came about because I'm dynamically generating my panels, like so:

```html
<ngb-accordion #acc ...>
    <ng-panel *ngFor="let ...; index as i" [id]="i.toString()">
        <ng-template ngbPanelHeader>
            <div (click)="acc.toggle(i.toString())"
```

There are multiple places in the header and content where I also pass `i` to my typescript backend.

### Link to minimally-working StackBlitz that reproduces the issue:

[https://stackblitz.com/edit/angular-erh2kd?file=src/app/accordion-basic.html](https://stackblitz.com/edit/angular-erh2kd?file=src/app/accordion-basic.html)

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 11.2.8

ng-bootstrap: 9.1.0

Bootstrap: 4.6.0

**Comment by peterblazejewicz** [comment]
Why do you think this is a bug? I think usually the intent behind props like ID is to reflect `Element.id` somewhere in the implementation. Say, NgBoostrap uses this directly in couple of places, e.g via host bindings:
```ts
'[id]': 'id'
```
The backing native `id` type is a string. 
For NgPanel there is no exlicit declaration of type, using type inference feature in TypeScript. 
```ts
  @Input() id = `ngb-panel-${nextId++}`;
```

**Comment by pkozlowski-opensource** [comment]
I do agree with @peterblazejewicz analysis but at the same time I think we can remedy the situation by using `getElementById` instead of doing `.querySelector('#' + panelId);` in https://github.com/ng-bootstrap/ng-bootstrap/blob/82e8b380c93cd79079329d1dc3cbc60a2c329e35/src/accordion/accordion.ts#L430

This should be a trivial PR, help welcomed.

**Comment by Atifarooq** [comment]
I have fixed it and generate PR, you guys can have look and approve it
https://github.com/ng-bootstrap/ng-bootstrap/pull/4312/

**Comment by maxokorokov** [comment]
Actually fixed by #4196
